prov/efa/src/efa_mr.c

```diff
@@ -305,6 +305,15 @@ static int efa_mr_dereg_impl(struct efa_mr *efa_mr)
 			ret = err;
 		}
 	}
+
+	if (efa_mr->peer.iface == FI_HMEM_CUDA && cuda_is_gdrcopy_enabled()) {
```

Comment:
I wonder if there is any value in allowing apps to specify gdrcopy per MR -- that might be needed for CUDA IPC. However, as the FI_HMEM API is defined today we can't do that.